packages/client/src/actions/actions.test.ts

```diff
@@ -143,12 +143,18 @@ describe("actions", () => {
     const clientBoundActionTakesAttachment = client(
       actionTakesAttachment,
     ).applyAction;
-    expectTypeOf<Parameters<typeof clientBoundActionTakesAttachment>[0]>()
```

Comment:
Sadly, after converting the params to take a `ReadonlyArray<X>` this started evaluating the second half of the union to `never[]`. 

Since we actually only need assignability in the opposite direction, I flipped it in the tests.